src/main/java/mate/jdbc/service/impl/DriverServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package mate.jdbc.service.impl;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.DriverService;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private static DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        return driverDao.get(id).orElseThrow(NoSuchElementException::new);
```

Comment:
```suggestion
        return driverDao.get(id).orElseThrow();
```
NoSuchElementException is default exception